.travis.yml

```diff
@@ -20,25 +20,25 @@ before_install:
   # We do this conditionally because it saves us some downloading if the
   # version is the same.
   - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
-      wget https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+      wget https://repo.continuum.io/miniconda/Miniconda2-4.3.21-Linux-x86_64.sh -O miniconda.sh;
```

Comment:
Looks like 4.3.27.1 was released with the fix for this particular issue, should we pin to that instead?

Reply:
just made that change